enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -0,0 +1,310 @@
+# Improved Support for Managing Operators through Policy
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+  [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Enhance the policy framework with a new kind, `OperatorPolicy`, to allow for better management of
+OLM Operator installations on managed clusters. The controller for this kind would be included in
+the configuration-policy-controller container, on managed clusters. The new controller would better
+handle imperative installation steps for OLM objects, and provide better support for upgrade and
+uninstallation scenarios.
```

Comment:
> The new controller would better handle imperative installation steps for OLM objects

I guess it is referring to the fact that you need to approve the installation when the install plan approval strategy has been set to manual in the subscription.
I would propose to rephrase to: "The new controller would better handle installation requiring approvals of OLM operators."
Technically it is possible to handle everything in a declarative way. At the end of the day the API is a resource, whose fields can be modified. I understand the challenge with InstallPlans that are a second level resources (not directly created) but need to be modified for the approval.